channel/consensus_channel/helpers_test.go

```diff
@@ -47,7 +47,7 @@ func makeOutcome(left, right Balance, guarantees ...Guarantee) LedgerOutcome {
 	for _, g := range guarantees {
 		mappedGuarantees[g.target] = g
 	}
-	return LedgerOutcome{left: left, right: right, guarantees: mappedGuarantees}
+	return LedgerOutcome{leader: left, follower: right, guarantees: mappedGuarantees}
```

Comment:
This jars as it mixes leader/follower with left/right. Do you need to change the input variable names, too?

Reply:
2f74e76a0cfbbdb